runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -332,6 +352,7 @@ private void addCounter()
 			capsString(taskName), points, streak));
 
 		infoBoxManager.addInfoBox(counter);
+		setInfoTimer(Instant.now());
```

Comment:
This is probably not the best place for this since it is only hit if the counter == null. It would be best to put this in killOne and also setTask